qa/tasks/ses_qa.py

```diff
@@ -142,6 +142,40 @@ def begin(self):
                 raise ConfigError(self.err_prefix + "No such method ->{}<-"
                                   .format(method_spec))
 
+    def drive_replace_initiate(self, **kwargs):
+        """
+        Initiate Deepsea drive replacement
+
+        Assumes there is 1 drive not being deployed (1node5disks - with DriveGroup `limit: 4`)
+
+        In order to "hide" an existing disk from the ceph.c_v in teuthology
+        the disk is formatted and mounted.
+        """
+        # :TODO: make this random and find correct disk device
+        osd_id = "2"
+        cmd = """sudo ceph osd tree -f json-pretty |
+                 jq '[.nodes[] | select(.type == \"osd\")] | length'"""
```

Comment:
you could move this `cmd = ...` up to the top of the file, and change `cmd` to `number_of_osds_in_cluster` (or `total_osds`?)

This would eliminate the code duplication below